pkgs/tools/audio/goxlr-utility/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, installShellFiles
+, pkg-config
+, libpulseaudio
+, dbus
+, speechd
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "goxlr-utility";
+  version = "v0.12.6";
+
+  src = fetchFromGitHub {
+    owner = "GoXLR-on-Linux";
+    repo = pname;
+    rev = version;
+    sha256 = "vvaKCsqncRhag8IrS0AIfNqNHGU2WIvFaYISEVfUB2Y=";
+  };
+
+  buildInputs = [
+    libpulseaudio
+    dbus
+    speechd
+  ];
+
+  nativeBuildInputs = [
+    pkg-config
+    installShellFiles
+    rustPlatform.bindgenHook
+  ];
+
+  postInstall = ''
+    install -Dm644 "50-goxlr.rules" "$out/etc/udev/rules.d/50-goxlr.rules"
+
+    install -Dm644 "daemon/resources/goxlr-utility.png" "$out/share/icons/hicolor/48x48/apps/goxlr-utility.png"
+    install -Dm644 "daemon/resources/goxlr-utility.svg" "$out/share/icons/hicolor/scalable/apps/goxlr-utility.svg"
+    install -Dm644 "daemon/resources/goxlr-utility-large.png" "$out/share/pixmaps/goxlr-utility.png"
+    install -Dm644 "daemon/resources/goxlr-utility.desktop" "$out/share/applications/goxlr-utility.desktop"
+    sed -i -e "s#/usr/bin#$out/bin#g" -e "s#goxlr-launcher#goxlr-daemon#" "$out/share/applications/goxlr-utility.desktop"
```

Comment:
Maybe it'd be better to use `substituteInPlace`? This way you'd get at least a warning if the substitution fails in future update:

```
substituteStream(): WARNING: pattern '$pattern' doesn't match anything in $description
```

Reply:
Yup, and also makes it compatible with BSD tooling as the BSD sed at least used to require `-i ""` instead of plain `-i`

Reply:
> Yup, and also makes it compatible with BSD tooling as the BSD sed at least used to require `-i ""` instead of plain `-i`

I don't think that matters inside the Nix sandbox :).